includes/Integrations/Wcfm/OrderMigrator.php

```diff
@@ -0,0 +1,416 @@
+                    foreach ( $refund_tax as $refund_tax_id => $refund_tax_price ) {
+                        $refunded_amount += (float) $refund_tax_price;
+                    }
+                }
+
+                try {
+                    $line_items[ $item_id ] = array(
+                        'refund_total' => $c_refunded_amount,
+                        'refund_tax'   => $refund_tax,
+                    );
+
+                    $line_items[ $item_id ]['qty'] = $c_refunded_qty;
+
+                    $wcfm_create_refund_args = array(
+                        'amount'         => round( $refunded_amount, 2 ),
+                        'reason'         => $refund_reason,
+                        'order_id'       => $child_order->get_id(),
+                        'line_items'     => $line_items,
+                        'restock_items'  => $restock_refunded_items,
+                    );
+
+                    // Create the refund object.
+                    $refund = '';
+                    if ( $from_suborder ) {
+                        $refund = wc_create_refund( $wcfm_create_refund_args );
+                    }
+
+                    if ( ( $from_suborder && ! is_wp_error( $refund ) || ! $from_suborder ) ) {
+                        $refund_status = $refund_info->refund_status === 'completed' ? true : false;
+
+                        $item_totals    = json_encode( [ $item_id => $c_refunded_amount ] );
+                        $c_refunded_qty = json_encode( [ $item_id => $c_refunded_qty ] );
+                        $this->dokan_sync_refund_table(
+                            $child_order->get_id(),
+                            $seller_id,
+                            round( $refunded_amount, 2 ),
+                            $refund_reason,
+                            $c_refunded_qty,
+                            $item_totals,
+                            $refund_tax_total,
+                            $restock_refunded_items,
+                            $refund_info->refund_paid_date,
+                            $refund_status,
+                            $this->order->get_payment_method()
+                        );
+                    }
+                } catch ( \Exception $e ) {
+                    error_log( print_r( $e->getMessage(), 1 ) );
+                }
+            }
+        }
+    }
+
+    /**
+     * Rename vendor shipping for an order
+     *
+     * @since DOKAN_MIG_SINCE
+     *
+     * @param object $order
+     *
+     * @return array
+     */
+    public function get_order_vendor_shipping() {
+        $shipping_items = $this->order->get_items( 'shipping' );
+
+        foreach ( $shipping_items as $shipping_item_id => $shipping_item ) {
+            $order_item_shipping = new \WC_Order_Item_Shipping( $shipping_item_id );
+            $shipping_vendor_id  = $order_item_shipping->get_meta( 'vendor_id', true );
+
+            ! $shipping_vendor_id ? $shipping_vendor_id = 0 : '';
```

Comment:
This shorthand is not a good practice. Expand this kind of shorthand.
```
if ( ! $shipping_vendor_id ) {
    $shipping_vendor_id = 0;
}
```
Also, this line should not be needed. I guess doing the following in the previous line will do the trick.
```
shipping_vendor_id = (int) $order_item_shipping->get_meta( 'vendor_id', true );
```